lib/components/user/places/place-editor.js

```diff
@@ -97,23 +100,24 @@ class PlaceEditor extends Component {
         </>
         )}
 
-        <div style={{ display: 'flex' }}>
+        <Flex>
```

Comment:
Maybe FlexContainer instead or just Container? I don't know why this concerns me but `<Flex>` seems dangerously to a component we might build in the future for GTFS-flex.

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/280/commits/436b93da0ef77b9c75628f92d29cf8313085d163.